### DISCUSSION TITLE
Sign for a transaction using greenfield + BTCPay Vault + hardware wallet without going to BTCPay Admin?

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#2819] [General]
I was wondering... is there a way to sign for a transaction using greenfield + BTCPay Vault + hardware wallet without going to BTCPay Admin? This would make sending coins from inside Magento (or other eCommerce backend) a possibility, like for refunds or to transfer coins to an exchange to convert to fiat later. I guess it would be doable if we have the right APIs/JavaScript/communication with Vault in Magento, right?

**Comment by Kukks** [comment]
> I was wondering... is there a way to sign for a transaction using greenfield + BTCPay Vault + hardware wallet without going to BTCPay Admin? This would make sending coins from inside Magento (or other eCommerce backend) a possibility, like for refunds or to transfer coins to an exchange to convert to fiat later. I guess it would be doable if we have the right APIs/JavaScript/communication with Vault in Magento, right?

Not yet possible, but we intend to expose the PSBT functionality inside of BTCPay that will allow you to sign and broadcast transactions without needing a hot wallet for API spending. 

Vault is app agnostic and is only an HTTP wrapper for HWI
